leaflet-treering.js

```diff
@@ -448,7 +452,7 @@ function MeasurementData (dataObject, Lt) {
 
     k++;
 
-    second_points.map(e => {
+    second_points.map((e, i) => {
```

Comment:
you added the index here but don't seem to be using it?

Reply:
Old solution idea. Forgot to remove it.